src/move/econia/sources/user.move

```diff
@@ -1751,8 +1751,9 @@ module econia::user {
             (size as u128) * (market_account_ref_mut.lot_size as u128);
         // Calculate ticks to fill order.
         let ticks = (size as u128) * (price as u128);
-        // Assert ticks to fill order is not too large.
-        assert!(ticks <= (HI_64 as u128), E_TICKS_OVERFLOW);
+        if (ticks > (HI_64 as u128)) {
+            ticks = (HI_64 as u128)
+        };
```

Comment:
Silent reassignment without feedback here will create an accounting error against the order book, which maintains its own copy of state. I suggest this be removed.